src/cli/commands/dev.ts

```diff
@@ -35,9 +35,33 @@ export class Dev implements Command {
 
     log.info('start', { version: ownPackage.version })
 
+    const layoutPlugin: Plugin.WorktimeHooks = {
```

Comment:
Something I think both of us want to see more and more, internal features implemented as plugins!

Reply:
💯!